opencti-platform/opencti-front/tests_e2e/settings/parameters.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test } from '@playwright/test';
+import { expect } from '../fixtures/baseFixtures';
+import { LeftBarPage } from '../model/menu/leftBar.pageModel';
+
+test('Check EE activation', async ({ page }) => {
+  const leftBarPage = new LeftBarPage(page);
+
+  await page.goto('/');
+  await page.getByTestId('ChevronRightIcon').click();
+  await leftBarPage.clickOnMenu('Settings', 'Parameters');
+
+  await page.getByRole('button', { name: 'Enable Enterprise Edition' }).click();
+  await expect(page.getByRole('heading', { name: 'OpenCTI Enterprise Edition (' })).toBeVisible();
+  await page.getByLabel('I have read and agree to the').check();
+  await page.getByRole('button', { name: 'Enable' }).click();
+  expect(page.locator('div').filter({ hasText: /^Enterprise$/ })).toBeVisible();
+  await page.getByRole('button', { name: 'Disable Enterprise Edition' }).click();
+  expect(page.locator('div').filter({ hasText: /^Community$/ })).toBeVisible();
+});
+
+test('Check Logo replacement', async ({ page }) => {
+  const leftBarPage = new LeftBarPage(page);
+
+  await page.goto('/');
+  await page.getByTestId('ChevronRightIcon').click();
+  await leftBarPage.clickOnMenu('Settings', 'Parameters');
+
+  let logoSrc = await page.getByRole('link', { name: 'logo' }).locator('img').getAttribute('src');
+  expect(logoSrc).toContain('static/images/logo');
+  await page
+    .locator('input[name="platform_theme_dark_logo"]')
+    .fill('https://www.google.com/images/branding/googlelogo/1x/googlelogo_light_color_272x92dp.png');
+  await page
+    .locator('input[name="platform_theme_dark_logo"]')
+    .press('Tab');
+  await page.waitForTimeout(1000);
```

Comment:
You should not need this. Playwright is waiting for stuff by default, if you need it it's probably that the next line of the test is not made the right way.

IMO split in two lines, first await the link and in a second line await for the img.

Reply:
Well actually that was a quick workaround. The issue is that the logo is always visible on screen, we just need to wait for the mutation to change the display to check if the new one is there. I don't want to spend time finding a better way as it is working for now

Reply:
Well ok 🙄